pubpub/pubpub
# feature: Discussions in the new pub footer (2/x)

This PR integrates pub-bottom discussions into our new Pub footer redesign. There is still work to be done here in re-styling these discussions to match the new mockups, but I'm going to break that off as a different PR because this is a good (and shippable) intermediate state.

_Test plan:_
See the fixture [here](http://localhost:9001/?path=/story/containers-pub-pubdocument-pubbottom-pubbottom--default).
- Check that filtering discussions by anchored/archived/label status works
- Try searching discussions
- Try sorting discussions
- Take a look at the empty states

_Screenshots:_
Discussions in the footer:
<img width="704" alt="Screen Shot 2019-08-21 at 3 50 04 PM" src="https://user-images.githubusercontent.com/2208769/63463767-dc4a3780-c42b-11e9-8210-eba8abe73bea.png">
Focus on search:
<img width="735" alt="Screen Shot 2019-08-21 at 3 50 48 PM" src="https://user-images.githubusercontent.com/2208769/63463766-dc4a3780-c42b-11e9-9690-c4844f0a714d.png">
Focus on filtering:
<img width="712" alt="Screen Shot 2019-08-21 at 3 51 26 PM" src="https://user-images.githubusercontent.com/2208769/63463764-dbb1a100-c42b-11e9-9e19-8356253740ab.png">
Focus on empty state:
<img width="704" alt="Screen Shot 2019-08-21 at 3 51 13 PM" src="https://user-images.githubusercontent.com/2208769/63463765-dbb1a100-c42b-11e9-90e6-748df7f8ffbe.png">
